corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -103,18 +108,18 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
       </div>
     </div>
     <% if (actions || isMultiSelect) { %>
-    <div class="case-list-actions">
-      <% if (isMultiSelect) { %>
-      <button type="button" class="btn btn-success btn-lg formplayer-request" disabled="true" id="multi-select-continue-btn">{% trans "Continue" %} <span id="multi-select-btn-text" class="badge"><%- selectedCaseIds.length %></span></button>
-      <% } %>
-      <% if (actions) { %>
-        <% _.each(actions, function(action, index) { %>
-        <div class="case-list-action-button btn-group formplayer-request">
-          <button type="button" class="btn btn-default btn-lg " data-index="<%- index %>"><%- action.text %></button>
-        </div>
-        <% }) %>
-      <% } %>
-    </div>
+      <div class="case-list-actions">
```

Comment:
Do you know if we have a standard for whether or not to indent html after knockout conditionals like this? I've seen it done both ways pretty frequently.